src/utils/index.ts

```diff
@@ -32,3 +34,28 @@ export function scrollToTop(): void {
     behavior: 'smooth',
   });
 }
+
+// GITHUB Qualifier for search
+export const GHQ = {
+  stringify: function (obj: Record<string, string | undefined>): string {
+    return Object.entries(obj).filter(([_,v]) => !isEmpty(v)).map((pair) => pair.join(':')).join(' ')
+  },
+
+  parse: function(value: string): Record<string, string> {
+    const isValid = /^\w+:\w+( \w+:\w+)*$/.test(value);
+    if(!isValid) return {};
+
+    const pairs = value.split(' ').map(pair => pair.trim().split(':'))
+    const res: Record<string, string> = {};
+    pairs.forEach(([k,v]) => {
+      res[k] = v
+    })
+    return res;
+  }
+}
+
+export function convertText2Emoji(text: string): string {
```

Comment:
Can we rename `convertText2Emoji` to `convertTextToEmoji`?

Reply:
Yeah! Fn renamed